docs/auth/autologout.md

```diff
@@ -0,0 +1,75 @@
+---
+id: autologout
+title: Auto Logout
+# prettier-ignore
+description: This section describes how to setup the Auto Logout mechanism in Backstage
+---
+
+This section describes how to setup the Auto Logout mechanism in Backstage in case your organization needs it.
+
+## Summary
+
+The Auto Logout feature is an optional added security to Backstage, designed to automatically log out users after a preconfigured duration of inactivity. This capability helps to effectively mitigate the risks associated with unauthorized access through abandoned sessions, especially in shared device scenarios.
+
+The Auto Logout mechanism actively tracks user activity such as mouse movements, clicks, key pressing, and taps. If the system detects no activity over a set time span (idle timeout), it invalidates the user session and redirects to the login page.
+_Inactive users_ are the ones that don't perform any action on the Backstage app or that are logged in but no Backstage tab is open in the browser.
+
+This feature is particularly beneficial if your application should comply with internal policies within your organization that may require automatic logout after a specific period of inactivity.
+
+This is how it looks like:
+
+![Auto logout Preview](../assets/auth/autologout-preview.png)
+
+## Quick start
+
+To enable and configure Auto Logout, you will need to add the `<AutoLogout>` component to your Backstage's instance entry point, located at `App.tsx`.
+
+Here's how to add it:
+
+```ts
+import { AutoLogoutProvider } from '@backstage/core-components';
```

Comment:
Looks like this import is not used in the example below and there's a similar case further down